src/main/target/OMNIBUSF4FW/target.h

```diff
@@ -0,0 +1,180 @@
+/*
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define USE_TARGET_CONFIG
+
+// OMNIBUSF4FW for Version 2
+// OMNIBUSF4FW1/OFW1 for Public Test Version
+// (Not a valid target, .mk file must be supplied by a user)
```

Comment:
If you change the conditionals to be like

```
#if defined(OMNIBUSF4FW)
[...]
#if defined(OMNIBUSF4FW1)
[...]
￼#else
[...]
#endif
#endif //OMNIBUSF4FW
```

then test users will be able to build for the test version with `make OMNIBUSF4FW OPTIONS=OMNIBUSF4FW1` without having to modify the repository.